lib/src/model/game/game_controller.dart

```diff
@@ -431,6 +438,10 @@ class GameController extends _$GameController {
         _socketEventVersion = fullEvent.socketEventVersion;
         _lastMoveTime = null;
 
+        ref
```

Comment:
here it should be `ref.read`, as per riverpod documentation. We only want to watch in widget `build()` methods or in other providers because we want them to react to changes.